src/main/resources/templates/trips/index.html

```diff
@@ -6,19 +6,19 @@
 
         <br>
         <br>
+        <h1>My Saved Trips</h1>
+        <div class="flex-container">
+            <div th:each="trip : ${trips}">
+                <a class="trip-card" th:text="${trip.tripName}" th:href=" '/trips/trip?tripId=' + ${trip.id} "></a>
+            </div>
+        </div>
 
-        <table class="all-trips">
-            <thead>
-            <tr>
-                <th>Trip Name</th>
-            </tr>
-            </thead>
+<!--        <table >-->
 
-            <tr th:each="trip : ${trips}">
-                <td th:text="${trip.tripName}"></td>
-                <td><a th:href=" '/trips/trip?tripId=' + ${trip.id} ">View Trip</a></td>
-            </tr>
-        </table>
+<!--            <tr th:each="trip : ${trips}">-->
+<!--                <td><a class="trip-card" th:text="${trip.tripName}" th:href=" '/trips/trip?tripId=' + ${trip.id} "></a></td>-->
```

Comment:
There is a chunk of commented out code here in trips/index. Is this code block safe to go ahead and delete?

Reply:
deleted!